app/src/main/res/values/colors.xml

```diff
@@ -7,4 +7,8 @@
     <color name="teal_700">#FF018786</color>
     <color name="black">#FF000000</color>
     <color name="white">#FFFFFFFF</color>
+    <color name="light_blue_400">#FF29B6F6</color>
+    <color name="light_blue_600">#FF039BE5</color>
+    <color name="gray_400">#FFBDBDBD</color>
+    <color name="gray_600">#FF757575</color>
```

Comment:
Are you using these colors anywhere? If not, we can delete them--I've elected to just store colors as constants in `Colors.kt`. Makes life easier, lol